src/main/java/core/basesyntax/exseption/InvalidAgeException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exseption;
+
+public class InvalidAgeException extends RuntimeException {
```

Comment:
If you want to use your custom exception create only one InvalidDataException and use it in all cases when you get invalid data😉